### PR TITLE
Update bookings-faq.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a question to the FAQ -How to change bookable attribute under Manage staff?

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3614

File: microsoft-365/bookings/bookings-faq.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Bookings is available and active by default for Microsoft 365 Business Premium,
 
 Your customers will only see the Bookings calendar that you publish online for the services, times, and staff that you choose to register.
 
+### How to change bookable attribute under Manage staff?
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-faq.md]:
```suggestion
### How do I change a bookable attribute under Manage staff?
```

File: microsoft-365/bookings/bookings-faq.md [hunk 2]
```diff
@@ -30,6 +30,10 @@ Bookings is available and active by default for Microsoft 365 Business Premium,
...
+
+When the staff was added, he would have received an email for accepting or rejecting the request. They can open the same email and select "Reject" . This will make him non bookable. We currently do not have a way to toggle this from inside the app.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-faq.md]:
```suggestion
When staff members were added, they would have received an email to accept or reject the request. They can open the same email and select "Reject" to make them non-bookable. We currently do not have a way to toggle this from inside the app.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**